test/integration/4_routes/Caves/delete.test.js

```diff
@@ -5,7 +5,7 @@ const AuthTokenService = require('../../AuthTokenService');
 describe('Cave features', () => {
   let adminToken;
   before(async () => {
-    adminToken = await AuthTokenService.getRawBearerAdminToken();
+    adminToken = await AuthTokenService.getRawBearerModeratorToken();
```

Comment:
Something is wrong here. Either rename the variable or change the token type no ?